pkg/server/api/middleware/authorization.go

```diff
@@ -55,12 +55,14 @@ func (m *authorizationMiddleware) Preprocess(ctx context.Context, methodName str
 		ctx = rpccontext.WithLogger(ctx, rpccontext.Logger(ctx).WithFields(fields))
 	}
 
+	logger := rpccontext.Logger(ctx)
+
 	var deniedDetails *types.PermissionDeniedDetails
 	ctx, allow, err := m.opaAuth(ctx, req, methodName)
```

Comment:
Can we also not alias the ctx variable? I think that would be a little more robust towards future refactors.

```
authedCtx, allow, err := m.opaAuth(ctx, req, methodName)
....
if allow {
    return authedCtx, nil
}
```